.buildkite/steps/tests.sh

```diff
@@ -11,11 +11,18 @@ branch="${BUILDKITE_BRANCH:-main}"
 IMAGE=$(buildkite-agent meta-data get "agent-image")
 export IMAGE
 
-gotestsum \
-  --junitfile "junit-${BUILDKITE_JOB_ID}.xml" \
+args=(
+  --junitfile "junit-${BUILDKITE_JOB_ID}.xml"
   -- \
   -count=1 \
   -failfast \
   -ldflags="-X ${package}.branch=${branch}" \
-  "$@" \
```

Comment:
Unfortunate to use this, but it's not used in CI. It's easy enough to run test locally through the `justfile` directly.